README.md

```diff
@@ -190,8 +182,8 @@ Helper module that makes call of `getTheme` easier in React components, so you c
    - [`noCache = false`] *(Boolean)* - Default `noCache` flag if there is no `props.themeNoCache` passed.
 
 ### Examples
-Assume we have a themeable Icon component. Default composition for it is `'replace'`, but Button overrides it with `'merge'` one. Also, Button picks for Icon only classnames from its own style with prefix `icon-`. In result Button will render bigger green Icon.
-
+Assume we have a themeable Icon component. Default composition for it is `'replace'` declared in the render method in Icon component, but Button overrides it with `'merge'` declared as themeCompose='merge' in Button component. Button will use the prefix `icon-` in the CSS declaration: buttonStyles.css to concatenate the matching Icon classnames in iconStyles.css.
+As a result using `'merge'`, Button will render the bigger green Icon during the merge declaration. e.g {large: "Icon_c Button_z"} Button large will replace Icon large css declaration.
```

Comment:
`Button large will replace Icon large css declaration`
In case of merge it doesn't replace, it adds own classname to large Icon

Reply:
// this works

In case of merge it doesn't replace, it adds own classname to large Icon

Reply:
// maybe this

Assume we have a themeable Icon component. Default composition for it is `'replace'`
declared in the render method in Icon component, but Button overrides it with `'merge'`
declared as themeCompose='merge' in Button component. Button will use the prefix `icon-` in own CSS declaration: buttonStyles.css to concatenate the matching Icon classnames in iconStyles.css.
As a result using `'merge'`, Button will render the bigger green Icon during the merge declaration by adding own classname to Icon's large selector definition.
(e.g {large: "Icon_c Button_z"})

Reply:
Good!